components/Navbar.vue

```diff
@@ -220,6 +220,7 @@ const lastScrollPosition = ref(0)
 const isBurgerMenuOpened = ref(false)
 const isMobile = ref(window.innerWidth < 1024)
 const isMobileWithoutTablet = ref(window.innerWidth < 768)
+const isTinyMobile = computed(() => useWindowSize().width.value < 480)
```

Comment:
in a follow up, i would use:
`const { width } = useWindowSize()`
and change `ref(window.innerWidth)` by `computed(() => width.value)`
or consider using Bulma [breakpoints](https://bulma.io/documentation/overview/responsiveness/#breakpoints)

Reply:
The `useWindowSize` has been split into a separate line. The 1024 and 768 breakpoints I will try to refactor into Bulma breakpoints in another PR